### DISCUSSION TITLE
Yii2 support

### DISCUSSION THREAD

**Discussion by paulvales** [sfx101/docker-stacks#94] [Ideas]
Please add Yii2 to marketplace :)

**Comment by nabad600** [comment]
Hello @paulvales , Yii2 has been added,
![image](https://user-images.githubusercontent.com/67466982/208837999-f75b046e-94ad-4a00-8fc7-522099e046e2.png)

Gratitude for selecting DECK. Thank you for your input. We'd be glad to assist.
